packages/app/requirements.txt

```diff
@@ -8,7 +8,7 @@ galaxy-web-stack
 kombu
 Beaker
 pykwalify
-pulsar-galaxy-lib==0.14.0.dev4
+pulsar-galaxy-lib==0.14.1
```

Comment:
I think we don't need to pin released versions in packages (as long as we don't have infrastructure to sync pins). I think the pin was because `0.14.0.dev4` wasn't released yet?

Reply:
That makes sense to me; in any event we don't want it pinned to `.0.dev4` I guess.  I'll drop the pin here.